src/components/UNSTABLE__LayoutModules/Layout.stories.js

```diff
@@ -65,6 +67,28 @@ export default {
       <>
         <UIShell />
 
+        <InlineNotification
+          actions={
+            <NotificationActionButton
+              onClick={() =>
+                window.open(
+                  'https://github.com/carbon-design-system/carbon/issues/7717',
+                  '_blank'
+                )
+              }
+            >
```

Comment:
Because [`NotificationActionButton` uses Carbon's `Button` under-the-hood](https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/Notification/Notification.js#L38-L45), we can pass it link attributes and avoid using JavaScript for navigation - For example:

```suggestion
            <NotificationActionButton
              href="https://github.com/carbon-design-system/carbon/issues/7717"
              rel="noopener noreferrer"
              target="_blank"
            >
```